### DISCUSSION TITLE
@serialport/bindings is deprecated and renamed

### DISCUSSION THREAD

**Discussion by GruszekG** [Koenkk/zigbee2mqtt#11725] [General]
I have tried to install the Zigbee2MQTT. I followed this instruction: [instalation](https://www.zigbee2mqtt.io/guide/installation/05_python_virtual_environment.html).
Installation failed on the last step: 
`npm ci`
with eror messages:
`1536 info lifecycle @serialport/bindings@9.2.8~install: @serialport/bindings@9.2.8
1537 verbose lifecycle @serialport/bindings@9.2.8~install: unsafe-perm in lifecycle true
1538 verbose lifecycle @serialport/bindings@9.2.8~install: PATH: /opt/zigbee2mqtt/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/node_modules/zigbee-herdsman/node_modules/@serialport/bindings/node_modules/.bin:/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/node_modules/zigbee-herdsman/node_modules/.bin:/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/node_modules/.bin:/opt/zigbee2mqtt/node_modules/.bin:/opt/zigbee2mqtt/lib/node_modules/.bin:/opt/zigbee2mqtt/bin:/opt/zigbee2mqtt/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games
1539 verbose lifecycle @serialport/bindings@9.2.8~install: CWD: /opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/node_modules/zigbee-herdsman/node_modules/@serialport/bindings
1540 silly lifecycle @serialport/bindings@9.2.8~install: Args: [ '-c',
1540 silly lifecycle   'prebuild-install --tag-prefix @serialport/bindings@ || node-gyp rebuild' ]
1541 silly lifecycle @serialport/bindings@9.2.8~install: Returned: code: 134  signal: null
1542 info lifecycle @serialport/bindings@9.2.8~install: Failed to exec install script
1543 verbose teardown shutting down workers.
1544 info teardown Done in 0.014s
1545 verbose stack Error: @serialport/bindings@9.2.8 install: `prebuild-install --tag-prefix @serialport/bindings@ || node-gyp rebuild`
1545 verbose stack Exit status 134
1545 verbose stack     at EventEmitter.<anonymous> (/opt/zigbee2mqtt/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
1545 verbose stack     at EventEmitter.emit (events.js:189:13)
1545 verbose stack     at ChildProcess.<anonymous> (/opt/zigbee2mqtt/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
1545 verbose stack     at ChildProcess.emit (events.js:189:13)
1545 verbose stack     at maybeClose (internal/child_process.js:970:16)
1545 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
1546 verbose pkgid @serialport/bindings@9.2.8
1547 verbose cwd /opt/zigbee2mqtt
1548 verbose Linux 4.19.94-ti-r70
1549 verbose argv "/opt/zigbee2mqtt/bin/node" "/opt/zigbee2mqtt/bin/npm" "ci"
1550 verbose node v10.15.1
1551 verbose npm  v6.4.1
1552 error code ELIFECYCLE
1553 error errno 134
1554 error @serialport/bindings@9.2.8 install: `prebuild-install --tag-prefix @serialport/bindings@ || node-gyp rebuild`
1554 error Exit status 134
1555 error Failed at the @serialport/bindings@9.2.8 install script.
1555 error This is probably not a problem with npm. There is likely additional logging output above.
1556 verbose exit [ 134, true ]`

When I google it out I found that @serialport/bindings is deprecated and renamed. [link](https://www.npmjs.com/package/@serialport/bindings)
I not familiar with java script at all. So maybe it will be a stupid question.
Do you think it is a reason why my installation failed?
Maybe some package dependencies should be updated to new package name?

**Comment by Koenkk** [comment]
@serialport/bindings-cpp only has version 10 which is broken:

- https://www.npmjs.com/package/@serialport/bindings-cpp
- https://github.com/serialport/node-serialport/issues/2438
- https://github.com/Koenkk/zigbee2mqtt/issues/11545
